app/src/main/java/com/google/android/sambadocumentsprovider/SambaConfiguration.java

```diff
@@ -162,9 +200,7 @@ public Void run(Void... params) throws IOException {
 
     @Override
     public void onSucceeded(Void result) {
-      if (mListener != null) {
```

Comment:
I added it because I added the case where it could be null, and the case is being removed in this change.